tests/integration/utils.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+# SPDX-License-Identifier: MIT
+
+import requests
+from requests_kerberos import HTTPKerberosAuth
+
+
+class Response:
```

Comment:
Consider using a [namedtuple](https://docs.python.org/2/library/collections.html#collections.namedtuple), unless you plan on adding more functionality to this class later.

Reply:
done, namedtuples are cool  :)

Reply:
They sure are! :)